### PR TITLE
Add the first of the teams files

### PR DESCRIPTION
**Pull request by tas50**
This adds teams files for chef-infra, chef-foundation, chef-workstation, knife, win32

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: teams/chef-foundation.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+[mixlib-archive](https://github.com/chef/mixlib-archive)
+[mixlib-authentication](https://github.com/chef/mixlib-authentication)
+[mixlib-cli](https://github.com/chef/mixlib-cli)
+[mixlib-config](https://github.com/chef/mixlib-config)
+[mixlib-install](https://github.com/chef/mixlib-install)
+[mixlib-log](https://github.com/chef/mixlib-log)
+[mixlib-shellout](https://github.com/chef/mixlib-shellout)
+[mixlib-versioning](https://github.com/chef/mixlib-versioning)
+[omnibus-software](https://github.com/chef/omnibus-software)
+[omnibus](https://github.com/chef/omnibus)
```
Comment (robbkidd) [inline teams/chef-foundation.md]: With omnibus repos in here, maybe some representation from Chef's Release Engineering? @scotthain?

Maybe also include `[omnibus-toolchain](https://github.com/chef/omnibus-toolchain)`

Comment (robbkidd) [reply teams/chef-foundation.md]: I also volunteer as a Reviewer having had my hands in `omnibus*`, `chef-telemetry`, and the occasional `mixlib`.

Comment (tas50) [reply teams/chef-foundation.md]: yep @scotthain should be in here if he wants.

Comment (scotthain) [reply teams/chef-foundation.md]: I'm fine with  being an approver/reviewer

**Comment by robbkidd** [comment]
Alphabetized.

We were never here.

File: teams/chef-foundation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Chef Foundation Team
+
+The Chef Foundation team is responsible for common libraries and applications used to support projects such as Chef and InSpec.
+
+### Project Owners
+
+Clinton Wolfe
+  - Github: [clintoncwolfe](https://github.com/clintoncwolfe)
+  - Slack: @btm
```
Comment (btm) [inline teams/chef-foundation.md]: clinton / btm?

Comment (tas50) [reply teams/chef-foundation.md]: yep

**Pull request merged**